name-registry/README.md

```diff
@@ -0,0 +1,55 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset=".docs/name-registry-logo-dark-theme.png">
+        <img alt="SwayApps Fundraiser Logo" width="400px" src=".docs/name-registry-logo-light-theme.png">
+    </picture>
+</p>
+
+## Overview
+
+The name registry application allows users to use human readable names instead of addresses to send and receive payments, making it easier to transfer cryptocurrency. 
+
+A name can be registered for the price of 1 unit of any asset per 100 seconds and it resolves to any Identity the owner sets.
+In this implementation the price is paid in the base asset on the Fuel network.
+
+Both the asset and the price per 100 seconds are configuration time constants so can be easily changed to different values.
+
+More information can be found in the [specification](./SPECIFICATION.md).
+
+## Project Structure
+
+The project consists of a smart contract and a user interface which the user can interact with.
+
+<!--Only show most important files e.g. script to run, build etc.-->
+
+```
+name-registry/
+├── contract/
+|    └── src/main.sw
+|    └── tests/harness.rs
+├── frontend/
+├── README.md
+└── SPECIFICATION.md
+```
+
+## Running the project
+
+### User Interface
+
+TODO: UI does not currently exist
+
+### Tests
+
+In order to run the tests make sure that you are in the root of this project i.e. `/path/to/name-registry/<you are here>`
+
+Run the tests
+   ```bash
+   forc build
+   cargo test
+   ```
```

Comment:
Once this PR goes in can you raise 1 issue which says that this change needs to be made for each specification? It should be labelled as a `Good First Issue`.